release notes/v1.2.0.md

```diff
@@ -0,0 +1,363 @@
+k6 v1.2.0 is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+## Breaking changes
+
+- The experimental Open Telemetry and Prometheus outputs are now enforcing TLSv1.3. This should've been the default to begin with. It is not expected that anyone should be affected, apart from making it more secure for the metrics output to send messages.
+
+
+## New features
+
+_optional intro here_
+
+### Binary Provisioning
+
+We have made multiple enhancements to the binary provisioning feature:
+
+- [#4811](https://github.com/grafana/k6/pull/4811) Removes limitations for processing test scripts from stdin or using file extensions other than `.js` or `.tar`.
+- [#4812](https://github.com/grafana/k6/pull/4812) Considers built-in extensions before attempting binary provisioning. When running a test using a custom `k6` binary, the binary provisioning checks if the binary includes all required extensions.
+' [#4921](https://github.com/grafana/k6/pull/4921) Removes the requirement for GCK6 authentication for binary provisioning, and enables the binary provisioning feature for local execution (i.e., the `k6 run` command).
+- [#4922](https://github.com/grafana/k6/pull/4922) Supports running tests using community extensions.
+
+Previously, k6 only supported official extensions. As of the PR [#4922](https://github.com/grafana/k6/pull/4922), it's now possible to use any extension listed in the [extensions registry](https://grafana.com/docs/k6/latest/extensions/explore/) by setting the `K6_ENABLE_COMMUNITY_EXTENSIONS` environment variable.
+
+```
+K6_BINARY_PROVISIONING=true K6_ENABLE_COMMUNITY_EXTENSIONS=true k6 run script.js
```

Comment:
```suggestion
K6_ENABLE_COMMUNITY_EXTENSIONS=true k6 run script.js
```
If we merge https://github.com/grafana/k6/pull/5014 then we will rewrite this section to remove the flag mentions.